docs/product/components/links.html

```diff
@@ -206,11 +206,11 @@
 <a class="s-block-link s-block-link__left is-selected" href="#">…</a>
 {% endhighlight %}
         <div class="stacks-preview--example">
-            <div class="d-flex gs16 fd-column ws3">
+            <div class="d-flex g16 fd-column ws3">
                 <a class="flex--item s-block-link s-block-link__left" href="#">Block link</a>
             </div>
 
-            <div class="d-flex gs16 fd-column ws3">
+            <div class="d-flex g16 fd-column ws3">
```

Comment:
## No action necessary

This renders slightly differently than what's in `develop`. Same as https://github.com/StackExchange/Stacks/pull/1157#discussion_r1003817900

<details>
<summary>Diff</summary>

![image](https://user-images.githubusercontent.com/647177/197642493-505e460a-a648-4bb1-ac4a-78885b5e9402.png)

</details>

Reply:
👍